python/tvm/relay/op/contrib/arm_compute_lib.py

```diff
@@ -114,8 +141,26 @@ def check_qnn_conv(extract):
             call = call.args[0]
         return qnn_conv2d(call.attrs, call.args)
 
+    def check_dense(extract):
+        """Check conv pattern is supported by ACL."""
+        call = extract
```

Comment:
After reading `check_qnn_dense`, do you need to check the `out_dtype` here?

Reply:
Good catch, thanks